include/itkMeshProcrustesAlignFilter.h

```diff
@@ -53,7 +53,7 @@ namespace itk
  * change  Martin Styner, UNC support for single template and initialization with average
  * TODO: Enable/Disable Normalization of centering and scaling to origin
  *
- * \ingroup Mesh3DProcrustesAlignFilter
+ * \ingroup ITKShape
```

Comment:
This might need to be `Shape` instead of `ITKShape`. Same for other instances below.

Reply:
Thanks @dzenanz . This one did indeed need to be changed, it turned out the last one I was overlooking was in `CMakeLists.txt` (of course).